pocket/chain.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pocket",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Network MainNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket",
+  "bech32_prefix": "pokt@",
```

Comment:
I presume this isn't meant to include the @ character? Highly unusual...

Reply:
Removed.